hi-ml-azure/src/health_azure/utils.py

```diff
@@ -1169,12 +1181,13 @@ def set_environment_variables_for_multi_node() -> None:
         return
 
     if ENV_MASTER_PORT not in os.environ:
-        os.environ[ENV_MASTER_PORT] = "6105"
+        os.environ[ENV_MASTER_PORT] = str(MASTER_PORT_DEFAULT)
 
     if ENV_OMPI_COMM_WORLD_RANK in os.environ:
         os.environ[ENV_NODE_RANK] = os.environ[ENV_OMPI_COMM_WORLD_RANK]  # node rank is the world_rank from mpi run
     env_vars = ", ".join(f"{var} = {os.environ[var]}" for var in [ENV_MASTER_ADDR, ENV_MASTER_PORT, ENV_NODE_RANK])
-    print(f"Distributed training: {env_vars}")
+    print(env_vars)
```

Comment:
Remove the print statement?

Reply:
I still see the print statement in line 1202, even though we print the same info to logging in the line below. Is that by accident or design?